internal/rbd/replicationcontrollerserver.go

```diff
@@ -368,8 +368,26 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 	ready := false
 	state := imageMirroringState(mirrorStatus.State)
 	if state == upAndStopped || state == upAndReplaying {
+		// Make sure the peer site image state is up and stopped
 		ready = true
+		for _, s := range mirrorStatus.PeerSites {
```

Comment:
`PeerSites` describes multiple sites, if there is peer in a incomplete state, you may want to log that peer so users can investigate what the cause could be

Reply:
Added logging now.